README.md

```diff
@@ -28,6 +28,12 @@
 
 [![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+**LATEST RELEASE: You are currently on the main branch which tracks
+under-development progress towards the next release. The current release branch
+is [r23.10](https://github.com/triton-inference-server/vllm_backend/tree/r23.10)
```

Comment:
so, we are keeping hard-coded value for the latest release?

Reply:
See the explanation above. I am open for other suggestions.